CMakeLists.txt

```diff
@@ -44,6 +44,14 @@ option( BUILD_PACKAGE "Build package" OFF )
 option( BUILD_TESTING "Build testing targets" OFF )
 option( BUILD_FORTRAN "Build Fortran stuff" OFF )
 
+# we don't need all devs to have Python installed.  There are 2 cases where we do want it
+# 1. Doing regression testing; which would be: BUILD_TESTING AND ENABLE_REGRESSION_TESTING
+# 2. Doing package building, since Python is going to be our tool for some things; thus: BUILD_PACKAGE
+
+if( ( BUILD_TESTING AND ENABLE_REGRESSION_TESTING ) OR ( BUILD_PACKAGE ) )
+  find_package(PythonInterp REQUIRED)
```

Comment:
Not a Pythonist but since you tell me there is a difference between Python 2 and 3, don't you think you should specify a version?